util/cpt_upgraders/riscv-vext.py

```diff
@@ -0,0 +1,59 @@
+def upgrader(cpt):
```

Comment:
This needs a license. Sorry for not noticing until now.

Reply:
done!